src/test/java/org/gridsuite/spreadsheetconfig/server/SpreadsheetConfigControllerTest.java

```diff
@@ -0,0 +1,294 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.spreadsheetconfig.server;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.gridsuite.spreadsheetconfig.server.constants.SheetType;
+import org.gridsuite.spreadsheetconfig.server.dto.CustomColumnInfos;
+import org.gridsuite.spreadsheetconfig.server.dto.MetadataInfos;
+import org.gridsuite.spreadsheetconfig.server.dto.SpreadsheetConfigInfos;
+import org.gridsuite.spreadsheetconfig.server.repositories.SpreadsheetConfigRepository;
+import org.gridsuite.spreadsheetconfig.server.service.SpreadsheetConfigService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * @author Achour BERRAHMA <achour.berrahma at rte-france.com>
+ */
+@SpringBootTest
+@AutoConfigureMockMvc
+class SpreadsheetConfigControllerTest {
```

Comment:
These are integration tests and not unit tests of the controller, so please rename the file SpeadsheetConfigIntegrationTest